crates/chain/src/request.rs

```diff
@@ -0,0 +1,32 @@
+use crate::collections::BTreeMap;
+use crate::local_chain::CheckPoint;
+use alloc::vec::Vec;
+use bitcoin::{OutPoint, ScriptBuf, Txid};
+
+/// Data required to perform a blockchain client sync.
```

Comment:
Imo we should be clear that these structures are only useful for spk-based syncing.

Reply:
I updated the docs.

Reply:
done.